lib/widgets/content.dart

```diff
@@ -107,6 +109,27 @@ class BlockContentList extends StatelessWidget {
   }
 }
 
+/// A Divider matching the style found in:
+/// [web/styles/rendered_markdown.css](https://github.com/zulip/zulip/blob/main/web/styles/rendered_markdown.css)
+class ThematicBreak extends StatelessWidget {
+  const ThematicBreak({
+    super.key,
+    this.thickness = 2,
+    this.color,
+  });
+
+  final double thickness;
+  final Color? color;
+
+  @override
+  Widget build(BuildContext context) {
+    return Divider(
+      color: color ?? const HSLColor.fromAHSL(1, 0, 0, .87).toColor(),
+      thickness: 2,
+    );
```

Comment:
Looking at the web app, I see that it puts more space above and below the line than we get with this. Let's match web, or else write down why we aren't doing so.